pkg/manifest/render/render.go

```diff
@@ -0,0 +1,274 @@
+package render
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"hash/fnv"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"k8s.io/klog"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/openshift/cluster-version-operator/pkg/manifest"
+)
+
+const (
+	DefaultPayloadDir = "/"
+
+	CVOManifestDir     = "manifests"
+	ReleaseManifestDir = "release-manifests"
+
+	cincinnatiJSONFile  = "release-metadata"
+	ImageReferencesFile = "image-references"
+)
+
+// Render renders all the manifests from /manifests to outputDir.
+func Render(outputDir, releaseImage string) error {
```

Comment:
`oc` will benefit from having the `Manifest` type and loading functions in the shared `library-go` space.  I'm not clear on why we want `Render` in library-go.  I guess `oc adm release new ...` will consume it?  Seems orthogonal to ConfigMap signatures though.

Reply:
It's original methods, Render and renderDir, are required to work with manifests and since it was already needed, it is where I stuck the manifest related methods pulled from payload (see https://github.com/openshift/cluster-version-operator/pull/309/files#diff-8fadc5d8bd084451d026d5bf128dad66)

Reply:
You're right, I totally confused the use of manifests here with the signatures we want to import and optionally save to file.